src/main/webapp/yarn.lock

```diff
@@ -7145,7 +7164,7 @@ fsevents@^2.1.2:
     bootstrap-daterangepicker: ^3.0.3
     browserslist: ^4.16.6
     chroma-js: ^1.3.4
-    chromedriver: 92.0.0
+    chromedriver: ^95.0.0
```

Comment:
We need to get this into development and pushed up through the branches.

Reply:
I agree!